contents/docs/api/capture.mdx

```diff
@@ -339,9 +339,9 @@ print(response)
 
 </MultiLanguage>
 
-## Events without person profiles
+## Anonymous events
 
-For backward compatibility, PostHog captures events with [person profiles](/docs/data/persons) by default. To capture events without them, add the `$process_person_profile` property to the event and set it to `false`.
+For backward compatibility, PostHog captures identified events by default. To capture anonymous events, add the `$process_person_profile` property to the event and set it to `false`.
```

Comment:
We are actually planning to change this default. Not sure if it's worth mentioning "for now"